### PR TITLE
logs for file download

### PR DESCRIPTION
**Pull request by rahmedzaizi**
## Changes in this PR
logs for file download
## JIRA ticket
[AYR-437](https://national-archives.atlassian.net/jira/software/projects/AYR/boards/66?assignee=60ba6c41f8a90b006f65b5fb&selectedIssue=AYR-437)
### After

- [ ] Requires env variable(s) to be updated

[AYR-437]: https://national-archives.atlassian.net/browse/AYR-437?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/main/routes.py [hunk 1]
```diff
@@ -659,7 +660,11 @@ def download_record(record_id: uuid.UUID):
             "Content-Disposition": "attachment;filename=" + download_filename
         },
     )
-
+    keycloak_openid = get_keycloak_instance_from_flask_config()
+    decoded_access_token = keycloak_openid.introspect(session["access_token"])
+    user_id = decoded_access_token["sub"]
+    log_download_data = {"user_id": user_id, "file": key}
+    print(json.dumps(log_download_data))
```
Comment (anthonyhashemi) [inline app/main/routes.py]: can we replace this with use of the configured logger;

with appropriate log level;

```python
current_app.logger.info('Some info message')
current_app.logger.debug('Some debug message')
current_app.logger.warning('Some warning message')
current_app.logger.error('Some error message')
```


```python
from flask import current_app
```

Comment (pgandhizaizi) [reply app/main/routes.py]: updated code to use current_app.logger.info as requested

File: app/main/routes.py [hunk 2]
```diff
@@ -659,7 +661,9 @@ def download_record(record_id: uuid.UUID):
             "Content-Disposition": "attachment;filename=" + download_filename
         },
     )
-
+    current_app.logger.info(
```
Comment (anthonyhashemi) [inline app/main/routes.py:664]: lets add a test to show we log this info, can probaly just add an assertion to an existing test for successful download

Comment (pgandhizaizi) [reply app/main/routes.py:664]: caplog used to capture logging details for test cases in test_download.py

**Review by anthonyhashemi — APPROVED** [review]

**Pull request merged**